services/models/deployment.go

```diff
@@ -5,6 +5,39 @@ import (
 	"k8s.io/api/autoscaling/v1"
 )
 
+type DeploymentList struct {
+	// Namespace where the deployments live in
+	// required: true
+	// example: bookinfo
+	Namespace Namespace `json:"namespace"`
+
+	// Deployments for a given namespace
+	// required: true
+	Deployments []DeploymentOverview `json:"deployments"`
+}
+
+type DeploymentOverview struct {
+	// Name of the deployment
+	// required: true
+	// example: reviews-v1
+	Name string `json:"name"`
+
+	// Number of desired replicas
+	// required: true
+	// example: 2
+	Replicas int32 `json:"replicas"`
```

Comment:
I'm not sure if we need these in DeploymentOverview, as it is also part of health and we will fetch health asynchronously. So I think you can remove Replicas, AvailableReplicas and UnavailableReplicas.

So at the moment it would leave just the Name, until we add validation info via another PR.